app/user/tests/test_views.py

```diff
@@ -79,7 +80,9 @@ def test_invalid_user_id(self) -> None:
         link = reverse(
             "email-verify",
             kwargs={
-                "uidb64": urlsafe_base64_encode(force_bytes("300")),
+                "encoded_pk": urlsafe_base64_encode(
+                    force_bytes("2573d942-c6b3-4f9c-8e54-04a32ddc175d")
```

Comment:
test when it is not a valid uuid also, test a string

Reply:
Alright